packages/http-server/src/server.ts

```diff
@@ -101,6 +101,7 @@ const replyHandler = <LoaderInput>(
       body,
     };
 
+    request.log.info({ input }, 'Request came in');
```

Comment:
Could this be something like 

```suggestion
    request.log.info({ input }, 'Received request GET /foo/bar');
```

Looking at the log output it seems like really funny wording, and it could also continue to be useful if we were to hide negotiator logs behind a verbose switch or something.

Reply:
The `input` contains the path and the verb already, so you'll see them in the final message.

Reply:
Ok, so assuming prism isnt completely broken and failing to talk to the negotiator, maybe we dont need this message?

Reply:
The reason is to make the indentation look cool, otherwise the user would see negotiator messages directly, which have a "level 1" of indentation, instead of 0. Does that make sense?